src/views/stack_view.rs

```diff
@@ -77,18 +83,22 @@ impl<T: View> ChildWrapper<T> {
     /// Returns a reference to the inner view
     pub fn get_inner(&self) -> &View {
         match *self {
-            ChildWrapper::Shadow(ref shadow) => shadow.get_inner().get_inner(),
-            ChildWrapper::Plain(ref layer) => layer.get_inner(),
+            ChildWrapper::Shadow(ref shadow) => shadow.get_inner(),
```

Comment:
I think we still want the absolute inner view here, so it would be `get_inner().get_inner()` (just like before this PR)?